### PR TITLE
Use Raw Config Strings

### PR DESCRIPTION
**Pull request by kalverra**

### CODE REVIEW

File: integration-tests/client/chainlink_config_builder.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	return c
+}
+
+// AddP2PNetworkingV2 adds defaults for V2 P2P networking (also enables LogPoller)
+func (c *ChainlinkConfigBuilder) AddP2PNetworkingV2() *ChainlinkConfigBuilder {
+	c.config.P2P.V2.Enabled = Pointer(true)
+	c.config.P2P.V2.ListenAddresses = &[]string{"0.0.0.0:6690"}
+	c.config.P2P.V2.AnnounceAddresses = &[]string{"0.0.0.0:6690"}
+
+	c.config.Feature.LogPoller = Pointer(true)
```
Comment (AnieeG) [inline integration-tests/client/chainlink_config_builder.go:89]: Lets have a separate method for Features

File: integration-tests/client/chainlink_config_builder.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+				WSURL:   wsURL,
+				HTTPURL: httpURL,
+			})
+		}
+
+		c.config.EVM = append(c.config.EVM, &v2.EVMConfig{
+			ChainID: utils.NewBigI(network.ChainID),
+			Enabled: Pointer(true),
+			Nodes:   nodes,
+			Chain: v2.Chain{
```
Comment (AnieeG) [inline integration-tests/client/chainlink_config_builder.go:64]: how about passing Chain as parameter and use a default for usual cases? CCIP tests would need some customizations here.

File: integration-tests/client/chainlink_config_builder.go [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink/core/assets"
+	v2 "github.com/smartcontractkit/chainlink/core/chains/evm/config/v2"
+	chainlinkConfig "github.com/smartcontractkit/chainlink/core/services/chainlink"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"github.com/smartcontractkit/chainlink/core/utils"
+)
+
+type ChainlinkConfigBuilder struct {
+	config *chainlinkConfig.Config `toml:"-"`
```
Comment (gheorghestrimtu) [inline integration-tests/client/chainlink_config_builder.go:19]: Can we make `config` public ?

Comment (gheorghestrimtu) [reply integration-tests/client/chainlink_config_builder.go:19]: Or I can use `RawConfig()`

File: integration-tests/client/chainlink_config_builder.go [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+func (c *ChainlinkConfigBuilder) MustTOML() string {
+	rawTOML, err := toml.Marshal(c.config)
+	if err != nil {
+		log.Fatal().Err(err).Msg("Error marshalling config TOML")
+	}
+	return fmt.Sprintf("%s", rawTOML)
+}
+
+// RawConfig returns the raw config object so you can modify it directly
+func (c *ChainlinkConfigBuilder) RawConfig() *chainlinkConfig.Config {
```
Comment (AnieeG) [inline integration-tests/client/chainlink_config_builder.go:146]: If I want to use the default config first and add some customization over it, how do I set ChainlinkConfigBuilder.config and then use `MustTOML`

**Review by AnieeG — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**